README.md

```diff
@@ -355,6 +362,75 @@ property in your domain data, this data will be reflect automatically in the Dat
 _Please note that removed Kogito Services will remove the protobuf files associated to it as well. This means that you won't
 be able to see the data through the Data Index anymore, although the data still persisted in Infinispan._
 
+## Kogito Jobs Service deployment
+
+Like Data Index, [Jobs Service](https://github.com/kiegroup/kogito-runtimes/wiki/Jobs-Service) can be deployed via Operator or CLI. 
+If persistence is required, the operator will also deploy an Infinispan server using Infinispan Operator.
+
+### Kogito Jobs Service installation
+
+There's a couple of ways to install the Jobs Service into your namespace using Kogito Operator.
+
+#### Installing the Kogito Jobs Service with the Kogito CLI
+
+If you have installed the [Kogito CLI](https://github.com/kiegroup/kogito-cloud-operator#kogito-cli), 
+run the following command to create the Kogito Jobs Service resource:
+
+``` 
+$ kogito install jobs-service -p my-project
+```
+
+There's some options to customize the Jobs Service deployment with CLI. 
+Run `kogito install jobs-service --help` to understand and set them according to your requirements.  
+
+#### Installing the Kogito Jobs Service with the Operator Catalog (OLM)
+
+If you are running on OpenShift 4.x, you can use the OperatorHub user interface to create the Kogito Jobs Service resource. 
+In the OpenShift Web Console, go to **Installed Operators** -> **Kogito Operator** -> **Kogito Jobs Service**. 
+Click **Create Kogito Jobs Service** and create a new resource as shown in the following example:
+
+```ỳaml
```

Comment:
small typo `y

Reply:
What an eye! I thought it was a dust in my screen.